sklearn/decomposition/_nmf.py

```diff
@@ -1165,6 +1165,13 @@ class NMF(TransformerMixin, BaseEstimator):
     max_iter : integer, default: 200
         Maximum number of iterations before timing out.
 
+    regularization : {'both', 'components', 'transformation', None}
```

Comment:
As discussed in the [contributing guidelines](https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation) the default value for optional arguments should also be listed.

Reply:
The default value is listed immediately in the line below, as displaying it in the same line breaks linting (79 chars)

Reply:
You are right, see the same discussion in #17739. Let's wait for suggestions from other reviewers.

Reply:
I think it is fine to have it below.